### PR TITLE
Tracing queries from LyoStore

### PR DESCRIPTION
**Pull request by jadelkhoury**
Tracing queries from LyoStore

## Checklist

- [ ] This PR adds an entry to the CHANGELOG. _See https://keepachangelog.com/en/1.0.0/ for instructions. Minor edits are exempt._
- [x] This PR was tested on at least one Lyo OSLC server or adds unit/integration tests.
- [x] This PR does NOT break the API

### CODE REVIEW

File: core/oslc4j-jena-provider/src/main/java/org/eclipse/lyo/oslc4j/provider/jena/AbstractOslcRdfXmlProvider.java [hunk 1]
```diff
@@ -146,6 +147,7 @@ private void writeObjectsTo(final Object[] objects, final OutputStream outputStr
 			writer.write(model,
 						 outputStream,
 						 null);
+            log.trace("writeObjectsTo");
```
Comment (berezovskyi) [inline core/oslc4j-jena-provider/src/main/java/org/eclipse/lyo/oslc4j/provider/jena/AbstractOslcRdfXmlProvider.java:150]: How about using nanoTime twice as described in https://www.baeldung.com/java-measure-elapsed-time

Comment (berezovskyi) [reply core/oslc4j-jena-provider/src/main/java/org/eclipse/lyo/oslc4j/provider/jena/AbstractOslcRdfXmlProvider.java:150]: Actually, I think Java 8+ `Instant` is even better.

**Review by berezovskyi — CHANGES_REQUESTED** [review]

File: core/oslc4j-jena-provider/src/main/java/org/eclipse/lyo/oslc4j/provider/jena/AbstractOslcRdfXmlProvider.java (hunk 1 above)
Comment (jadelkhoury) [reply core/oslc4j-jena-provider/src/main/java/org/eclipse/lyo/oslc4j/provider/jena/AbstractOslcRdfXmlProvider.java]: done.

**Review by berezovskyi — APPROVED** [review]
Would be better with {} ms units but ok to merge if you want.

**Pull request merged**